### ISSUE TITLE
CommandAPI Preprocessor doesn't import CommandPermission in compiled class

### ISSUE THREAD

**Issue by Cr4zy5kyU** [JorelAli/CommandAPI#177]
**CommandAPI version**
`commandapi-shade:5.3`

**Minecraft version**
No Minecraft version involved

**Describe the bug**
CommandAPI Preprocessor in dependency `commandapi-shade:5.3` doesn't import class CommandPermission when it needs to.

**My code**
This code reproduces the behaviour:
```java
@NeedsOp
@Command("tryit")
public class UhcCommand {

	@Default
	private static void create(Player player) throws WrapperCommandSyntaxException {
		CommandAPI.fail("Fail-time!");
	}

}
```

**Expected behaviour**
Preprocessor to successfully compile the class, so maven can finish build.

**Screenshots**
[Generated class with error message](https://i.imgur.com/cZcMW9Z.png)
[Pom](https://imgur.com/a/koMS1p8)

**Additional context**
IDE: IntelliJ IDEA 2020.2.4 (Ultimate Edition) - Build #IU-202.8194.7

**Comment by JorelAli** [comment]
Fixed in commit 528a9283d2912955958464933567c496c3f3eb4a, scheduled for release v5.5.

**Comment by JorelAli** [comment]
@Cr4zy5kyU [CommandAPI v5.5 has been released](https://github.com/JorelAli/CommandAPI/releases/tag/v5.5).
